raiden/network/transport/matrix/utils.py

```diff
@@ -22,7 +22,7 @@
 
 log = structlog.get_logger(__name__)
 
-JOIN_RETRIES = 5
+JOIN_RETRIES = 100
```

Comment:
Wow, was that the necessary number to get the test to pass?

Reply:
Yes, otherwise it fails on travis..

Reply:
This is down to 10 now.